MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -1114,6 +1116,13 @@ void EnggDiffractionPresenter::doFocusing(const EnggDiffCalibSettings &cs,
   }
   g_log.notice() << "Saved focused workspace as file: " << fullFilename
                  << std::endl;
+
+  bool saveOutputFiles = m_view->saveOutputFiles();
+  if (saveOutputFiles) {
```

Comment:
I think they don't want/need the nexus file. Am I right or did Saurabh mention that as well?
In principle they want the `.his` (OpenGenie), `.gss` (GSAS), and `.dat` (XYE).